agent/ecscni/plugin_test.go

```diff
@@ -30,7 +30,7 @@ const (
 	// ECSCNIVersion, ECSCNIGitHash, VPCCNIGitHash needs to be updated every time CNI plugin is updated.
 	currentECSCNIVersion = "2020.09.0"
 	currentECSCNIGitHash = "55b2ae77ee0bf22321b14f2d4ebbcc04f77322e1"
-	currentVPCCNIGitHash = "199bfc65cced4951cbb6a38e6e828afa8c2b023c"
+	currentVPCCNIGitHash = "c3a89823446f312edfbed8375a0394773e39f807"
```

Comment:
24d6bd87707d1b1801086fc507ebab8d32067412 this is the latest after `Adding a new mode Tproxy to do the redirection of traffic for ServiceConnect Plugin (#74)`

Reply:
The latest CNI plugin repo commit updated request model which is incompatible with the code in this PR. In my follow-up PR for bridge mode, both the model and the CNI git hash will be updated. I wanted to keep PR to be a simple rebase if possible.